module-apps/application-call/include/application-call/ApplicationCall.hpp

```diff
@@ -17,7 +17,7 @@
 namespace app
 {
     inline constexpr auto name_call       = "ApplicationCall";
-    inline constexpr auto call_stack_size = 8192U;
+    inline constexpr auto call_stack_size = 8192 * 2U;
```

Comment:
please remember to double-check the stack requirements

Reply:
done